assets/base.css

```diff
@@ -2192,6 +2192,36 @@ input[type='checkbox'] {
   letter-spacing: 0.1rem;
 }
 
+.announcement-bar__message--small {
+  font-size: calc(var(--font-heading-scale) * 1.2rem);
```

Comment:
I thought we would call out the headings here to get their own property of letter-spacing and line-height. I would like to learn what is the most ideal scenario for learning/coaching purposes! 😇

Reply:
Ah yes. I added them in via https://github.com/Shopify/dawn/pull/1778/commits/e0fb64171b10fd4a02c336e567e4812c03c62a6b. 

The best practice (in my opinion) here would be to implement those values — as well as all of our standard font sizes — as CSS variables, so they're stored in a single place and we're not duplicating so much code.